starlite/response.py

```diff
@@ -108,7 +108,7 @@ def __init__(
         template_name: str,
         template_engine: "TemplateEngineProtocol",
         status_code: int,
-        context: Optional[Dict[str, Any]] = None,
+        context: Dict[str, Any],
```

Comment:
why is this required?

Reply:
At the point context is guaranteed to be a dict with `request` so removing `Optional` is just making typing a bit more strict.